docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -1,25 +1,22 @@
-# The B2B Edition of BigCommerce
+# B2B Edition 
 
-BundleB2B (B3) adds business-to-business (B2B) functionality to the BigCommerce platform, allowing businesses to easily facilitate B2B operations online. B3 provides a comprehensive suite of key B2B features to improve the B2B self-service experience for BigCommerce store owners and their customers.
+The BigCommerce platform offers BundleB2B (B3) functionality which allows businesses to streamline online operations and improve experience for BigCommerce store owners and their customers.
 
-The B2B Edition of BigCommerce is a bundled offering of the BigCommerce Enterprise solution, B3, and six BigCommerce themes optimized for B3.
+The B2B Edition is a packaged offering of the BigCommerce Enterprise plan, B3, and [six BigCommerce themes optimized for B3](https://support.bigcommerce.com/s/article/B2B-Edition?language=en_US#themes).
 
-B3 offers multiple enhancements to your BigCommerce store, including:
+B3 offers multiple B2B feature enhancements to your BigCommerce store, including the following:
 
-- Corporate account management
-- Sales representative masquerade
-- Shared shopping lists
-- Payment method visibility control
-- Company address book management
-- Sales representative quoting
-- Buy again
-- Trade professional application
+- Customer account hierarchy with user roles and permissions
+- Shared shopping lists and buy again functionality
+- Sales representative enablement tools
+- Quote management
+- Invoice & payment management
 
-## B3 integration with BigCommerce
+## Installing B3 
```

Comment:
I don't think anything about this section changed as a result of BC acquiring BundleB2B. This section doesn't really describe how to install B3, and the parts about the dashboard being an iFrame while the client portion is dynamically loaded are important to the rest of the doc.